CONTRIBUTING.md

```diff
@@ -0,0 +1,182 @@
+# Contributing to CoronaTracker
+
+## Table of Contents
+
+<!-- TOC -->
+
+- [About the Project](#about-the-project)
+  - [Project Values](#project-values)
+  - [Townhall Meetings](#townhall-meetings)
+- [How to Contribute](#how-to-contribute)
+  - [Reporting Bugs](#reporting-bugs)
+  - [Suggesting Features / Enhancements](#suggesting-features--enhancements)
+  - [Contributing Code](#contributing-code)
+  - [Staying Up to Date](#staying-up-to-date)
+- [Style Guides](#style-guides)
+  - [Naming Git Branches](#naming-git-branches)
+  - [Creating Git Commit Messages](#creating-git-commit-messages)
+  - [Front End Style Guide](#front-end-style-guide)
+- [Getting Support](#getting-support)
+
+<!-- /TOC -->
+
+## About the Project
+
+❤️ By the community, for the community. ❤️
+
+We are building an open source progressive web application to help reduce the number of unnecessary hospital visits, educate the public on facts vs myths, and provide a health and well-being chart for logging their journey during the Coronavirus (COVID-19) pandemic.
+
+### Project Values
+
+Our mission is to develop social good through technology. We value initiative, collaboration, experimentation, and integrity.
+
+We are a welcoming, warm, and collaborative group of people from all over the world! Here you can experiment and take chances.
+
+To FAIL is to make your First Attempt In Learning, and we are here to help. We encourage developers to learn new concepts and best practices while collaborating on a project that is scaling to help mankind against COVID-19.
+
+### Townhall Meetings
+
+We hold weekly townhall meetings on Saturdays, from 12:00pm EST to 12:30pm EST.
+
+The typical agenda is:
+
+1. Team leads will give an update
+2. Discuss project roadmap
+3. Discuss project deadlines
+4. Discuss action items for current goals
+
```

Comment:
Thinking about it, maybe we should add that there's time for contributors to participate and ask questions - partly because we do want to encourage participation, but secondly because having a lot of people on these calls leads to a lot of great opinions

Reply:
Fixed per c7bdd41, added "5. Time for community feedback and questions"